docs/getting-started/installation.md

```
@@ -161,32 +165,57 @@ Access Dagster UI:
-phlo materialize dlt_glucose_entries
+phlo create-workflow
 ```
 
-Or use the Dagster UI to materialize assets.
+Or materialize the example assets (if using a template with examples):
+
+```bash
+phlo materialize --all
+```
+
+You can also use the Dagster UI at http://localhost:10006 to materialize assets.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)